packages/multi-entry/README.md

```diff
@@ -0,0 +1,125 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-multi-entry
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-multi-entry
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-multi-entry
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-multi-entry
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-multi-entry
+
+🍣 A Rollup plugin which allows use of multiple entry points for a bundle.
+
+As an added bonus, the _named exports_ from all entry points will be combined. This is particularly useful for tests, but can also be used to package a library.
+
+_Note: `default` exports cannot be combined and exported by this plugin. Only named exports will be exported._
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v8.0.0+) and Rollup v1.20.0+.
+
+## Install
+
+Using npm:
+
+```console
+npm install @rollup/plugin-multi-entry --save-dev
+```
+
+## Usage
+
+Suppose that we have three separate source files, each with their own export(s):
+
+```js
+// batman.js
+export const belt = 'utility';
+```
+
+```js
+// robin.js
+export const tights = 'tight';
+```
+
+```js
+// joker.js
+export const color = 'purple';
+```
+
+Then, create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
+
+```js
+import multi from '@rollup/plugin-multi-entry';
+
+export default {
+  input: ['batman.js', 'robin.js', 'joker.js'],
+  output: {
+    dir: 'output'
+  },
+  plugins: [multi()]
```

Comment:
in here the plugin is referred to as `multi` but in an example a little bit later it's referred to as `multiEntry` - I think it would be best to unify those (preferably to `multiEntry`)

Reply:
agreed. unifying is best. typically we'll choose the terser name. ala node-resolve and auto-install.